From fc20ed0730573768130f6ebd4253ac7674f69d59 Mon Sep 17 00:00:00 2001
From: Zhong Lufan <lufanzhong@gmail.com>
Date: Sun, 6 Sep 2020 20:57:45 +0800
Subject: [PATCH] arm64: fix text offset for amlogic

---
 arch/arm64/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/Makefile b/arch/arm64/Makefile
index 70f590595..b669c93ce 100644
--- a/arch/arm64/Makefile
+++ b/arch/arm64/Makefile
@@ -121,7 +121,7 @@ endif
 head-y		:= arch/arm64/kernel/head.o
 
 # The byte offset of the kernel image in RAM from the start of RAM.
-TEXT_OFFSET := 0x0
+TEXT_OFFSET := 0x01080000
 
 ifeq ($(CONFIG_KASAN_SW_TAGS), y)
 KASAN_SHADOW_SCALE_SHIFT := 4
-- 
2.28.0

